crypto/random/seed/GetEntropyRandomSeed.c

```diff
@@ -24,7 +24,7 @@
 #define GetEntropyRandomSeed_GLIBC_HAS_IT \
     (defined(__GNU_LIBRARY__) && defined(__GLIBC_PREREQ) && __GLIBC_PREREQ(2, 25))
 
-#if !defined(__OPENBSD__) && !GetEntropyRandomSeed_GLIBC_HAS_IT
+#if !defined(__OPENBSD__) && !defined(GetEntropyRandomSeed_GLIBC_HAS_IT)
```

Comment:
`defined(GetEntropyRandomSeed_GLIBC_HAS_IT)` is always true, see two lines above. You might be looking for

```c
#if !defined(__OPENBSD__) && !(defined(GetEntropyRandomSeed_GLIBC_HAS_IT) && GetEntropyRandomSeed_GLIBC_HAS_IT)
```